test/system/037-runlabel.bats

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "podman container runlabel test" {
+    tmpdir=$PODMAN_TMPDIR/runlabel-test
+    mkdir -p $tmpdir
+    containerfile=$tmpdir/Containerfile
+    cat >$containerfile <<EOF
+FROM $IMAGE
+LABEL  INSTALL  /usr/bin/podman  run  -t  -i  --rm  \\\${OPT1}  --privileged  -v /:/host --net=host --ipc=host --pid=host -e HOST=/host -e NAME=\\\${NAME} -e IMAGE=\\\${IMAGE} -e CONFDIR=/etc/\\\${NAME} -e LOGDIR=/var/log/\\\${NAME} -e DATADIR=/var/lib/\\\${NAME} \\\${IMAGE} \\\${OPT2} /bin/install.sh \\\${OPT3}
+EOF
+
+    run_podman build -t runlabel_image $tmpdir
+
+    run_podman container runlabel --opt1=OPT1 --opt2=OPT2 --opt3=OPT3 --name test1 --display  install runlabel_image
```

Comment:
Just a quick once-over, but this caught my attention. Could I suggest:
```bash
    opt1=opt1$(random_string 8)
    optd=opt2$(random_string 10)
    --opt1=$opt1 --opt2=$opt2
```
Or, at a minimum, `--opt1=AAA --opt2=BBB`? Seeing them as `OPT1`, `OPT2` makes me have to triple-check the quoting in the Containerfile, make sure that `OPT1` isn't appearing in the results simply as a literal.